templates/data-provider/search.jsx

```diff
@@ -44,8 +53,8 @@ const DataProviderOutputsSearch = ({
       className={className}
     >
       {suggestions.map((el) => (
-        <Select.Option key={el.id} id={el.id} value={el.name} icon="#magnify">
-          {el.name}
+        <Select.Option key={el.id} id={el.id} value={el.title} icon="#magnify">
```

Comment:
Not sure if we need suggestions at the moment. Let me know.

Reply:
It doesn't look good at the moment. I would rather remove it.